functions.php

```diff
@@ -1,284 +1,290 @@
- * @since Simply Pure 1.0
- * @author Ritesh Sanap <riteshsanap@gmail.com>
- * 
- * @param string $title Default title text for current view.
- * @param string $sep Optional separator.
- * @return string The filtered title.
- */
-
-function purecss_wp_title( $title, $sep ) {
-	global $paged, $page;
-
-	if ( is_feed() ) {
-		return $title;
-	}
-
-	// Add the site name.
-	$title .= get_bloginfo( 'name', 'display' );
-
-	// Add the site description for the home/front page.
-	$site_description = get_bloginfo( 'description', 'display' );
-	if ( $site_description && ( is_home() || is_front_page() ) ) {
-		$title = "$title $sep $site_description";
-	}
-
-	// Add a page number if necessary.
-	if ( $paged >= 2 || $page >= 2 ) {
-		$title = "$title $sep " . sprintf( __( 'Page %s', 'purecss' ), max( $paged, $page ) );
-	}
-
-	return $title;
-}
-add_filter( 'wp_title', 'purecss_wp_title', 10, 2 );
-/**
- * Counts the number of sidebars with atleast a Single Widget
- * for more information visit : 
- * http://codex.wordpress.org/Function_Reference/is_active_sidebar
- *
- * @version 1.0
- * @since   Simply Pure 1.0
- * @author Ritesh Sanap <riteshsanap@gmail.com>
- *
- * @param   Array   $sidebars the List of sidebars that are active
- * @return  integer
- */
-function pure_sidebar_active_count($sidebars) {
-	$count = 0;
-	foreach ($sidebars as $sidebar) {
-		if(is_active_sidebar($sidebar)) {
-			$count++;
-		}
-	}
-	return $count;
-}
-/**
- * Class output for Footer and Post Footer widgets
- *
- * @version 1.0
- * @since   Simply Pure 1.0
- * @author Ritesh Sanap <riteshsanap@gmail.com>
- *
- * @param   string   $position
- * @return  string
- */
-function pure_sidebars_class($position) {
-	$sidebars = array();
-	if($position == 'post-footer') {
-		$sidebars = array('post-footer','post-footer-2','post-footer-3');
-	} else {
-		$sidebars = array('footer-area', 'footer-area-2', 'footer-area-3');
-	}
-
-	$class = 'pure-u-1';
-	$class .= ' pure-u-sm-1-'. pure_sidebar_active_count($sidebars);
-	return $class;
-}
-function pure_body_classes($classes) {
-	if(get_theme_mod('sidebar_position') != 'right') {
-		$classes[] = 'sidebar-left';
-	}
-	if(!is_active_sidebar('sidebar-area')) {
-		$classes[] = 'header-full';
-	} else {
-		$classes[] = 'header-half';
-	}
-
-	if(!get_theme_mod('display_avatar', true)) {
-		$classes[] = 'post-title-no-avatar';
-	}
-	return $classes;
-}
-add_filter( 'body_class', 'pure_body_classes' );
-require get_template_directory(). '/customizer.php';
-
-function purecss_archive_title() {
-	$title = NULL;
-	if(is_archive()) { 
-		$title = 'Archive: '. single_month_title(' ', false);
-	}
-	if(is_category()) {
-		  $title = 'Category: '. single_cat_title('', false); 
-	}
-	if(is_tag()) {
-		 $title = 'Tag: '. single_tag_title('', false);
-	}
-	if(is_author()) {
-		$title = 'Author: '. get_the_author();
-	}
-
-	return $title;
-}
\ No newline at end of file
+<?php 
+
+load_theme_textdomain('purecss', get_template_directory() . '/languages');

```

Comment:
No need to use load_theme_textdomain() 

instead check here 
https://developer.wordpress.org/plugins/internationalization/how-to-internationalize-your-plugin/